source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -111,6 +116,22 @@ export class Versions {
     static readonly v1_5 = new Version(1, 5);
     static readonly v1_6 = new Version(1, 6);
     static readonly latest = Versions.v1_6;
+    // version over latest is considered "vNext feature"
+    // if latest or maxSupportedVersion is changed,
+    // don't forget to update styles .ac-schema-version-1-?::after to add/remove Preview tag from adaptivecards-site\themes\adaptivecards\source\css\style.css
+    static readonly maxSupportedVersion = Versions.v1_6;
+}
+
+const LogEventIfVersionIsPreview = (sender: SerializableObject, name: string, value: any, context: BaseSerializationContext, version: Version) => {
```

Comment:
Why is this declared as a const equal to a lambda instead of simply a function like everywhere else in the codebase?

Also, why is it not a method on `BaseSerializationContext`?

Reply:
I can't remember where I read(possible react), arrow function can avoid 'this' binding problem in a class. And that's why I'm used to code with lambda.

I agree with you it's better to move it to BaseSerializationContext

Reply:
That is only when you set an event handler within the method of a class. Your `const` is global, there's not `this` to resolve in that context.

The below will not work:

```typescript
class MyClass {
    private _privateFields: string;

    someMethod() {
        let button = document.createElement("button");

        button.onclick = function() {
            this._privateField = "Private field set";
        }
    }
}
```

But this will:
```typescript
class MyClass {
    private _privateFields: string;

    someMethod() {
        let button = document.createElement("button");

        button.onclick = () => {
            this._privateField = "Private field set";
        }
    }
}
```

Reply:
not need it anymore